llpc/context/llpcPipelineContext.cpp

```diff
@@ -550,6 +550,7 @@ void PipelineContext::setUserDataNodesTable(Pipeline *pipeline, ArrayRef<Resourc
 
     destNode.sizeInDwords = node.sizeInDwords;
     destNode.offsetInDwords = node.offsetInDwords;
+    destNode.matchType = ResourceNodeType::Unknown;
```

Comment:
I think it would be best to always have an appropriate value in `matchType` (or whatever it gets renamed to).  Why not have the `matchType` contain `ResourceMappingNodeType::DescriptorTableVaPtr` when it is a descriptor table?